widget.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <version>0.0.1</version>
+        <id>SampleWidget</id>
+        <authors>SampleDev</authors>
+        <projectUrl>https://github.com/UiPath/assistant-plugin-creator</projectUrl>
+		    <description>A widget for UiPath Assistant</description>
+        <tags>Sample UiPath Assistant Widget</tags>
+        <dependencies>
+        </dependencies>
+    </metadata>
+
+    <files>
+		<file src="dist\widget.js" />
```

Comment:
Tabs used here, as well.
![image](https://user-images.githubusercontent.com/24717857/105326478-750f5480-5bd6-11eb-9133-b38f4eb8a899.png)

And also, I think this line should be indented once more.